File: QuickBind/src/main/java/com/wpf/app/quickbind/annotations/BindD2VHHelper.java
package com.wpf.app.quickbind.annotations;

import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.recyclerview.widget.RecyclerView;

import com.wpf.app.quick.annotations.BindD2VHelper;

/**
 * Created by 王朋飞 on 2022/7/6.
 */
public interface BindD2VHHelper<V extends View, Data> extends BindD2VHelper<RecyclerView.ViewHolder, V, Data> {

    void initView(@Nullable RecyclerView.ViewHolder viewHolder, @NonNull V view, @NonNull Data data);
}

File: QuickBind/src/main/java/com/wpf/app/quickbind/annotations/BindD2VHHelper.kt
package com.wpf.app.quickbind.annotations

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import com.wpf.app.quick.annotations.BindD2VHelper

/**
 * Created by 王朋飞 on 2022/9/5.
 *
 */
interface BindD2VHHelper<V : View, Data : Any> : BindD2VHelper<RecyclerView.ViewHolder, V, Data> {

    override fun initView(viewHolder: RecyclerView.ViewHolder?, view: V, data: Data)
}